add_to_app/multiple_flutters/README.md

```diff
@@ -0,0 +1,72 @@
+# multiple_flutters
+
+This is a sample that shows how to use the Flutter Engine Groups API to embed
+multiple instances of Flutter into an existing Android or iOS project.
+
+## Getting Started
+
+For iOS instructions, see:
+[multiple_flutters_ios](../multiple_flutters_ios/README.md).
+
+For Android instructions, see:
+[multiple_flutters_android](../multiple_flutters_android/README.md).
+
+## Flutter Engine Group
+
+These examples use the Flutter Engine Group APIs on the host platform which
+allows engines to share memory and CPU intensive resources. This leads to easier
+embedding of Flutter into an existing app since multiple entrypoints can be
+maintained via a FlutterFragment on Android or a UIViewController on iOS.
+Before FlutterEngineGroup, users had to juggle the usage of a small number of
+engines judiciously.
+
+More info on those API's can be found in the source
+code:
+
+- iOS -
+  [FlutterEngineGroup.h](https://github.com/flutter/engine/blob/master/shell/platform/darwin/ios/framework/Headers/FlutterEngineGroup.h)
+- Android -
+  [FlutterEngineGroup.java](https://github.com/flutter/engine/blob/master/shell/platform/android/io/flutter/embedding/engine/FlutterEngineGroup.java)
+
+## Important Files
+
+### iOS
+
+- [DataModel.swift](./multiple_flutters_ios/MultipleFluttersIos/HostViewController.swift)
+  — Observable data model that is the source of truth for the host platform and Flutter.
+- [HostViewController.swift](./multiple_flutters_ios/MultipleFluttersIos/HostViewController.swift)
+  — A UIViewController that synchronizes to the DataModel.
+- [main.dart](./multiple_flutters_module/lib/main.dart) — The Flutter source
+  code.
+- [SingleFlutterViewController.swift](./multiple_flutters_ios/MultipleFluttersIos/SingleFlutterViewController.swift)
+  — A subclass of FlutterViewController that synchronizes with the DataModel.
+- [DoubleFlutterViewController.swift](./multiple_flutters_ios/MultipleFluttersIos/DoubleFlutterViewController.swift)
+  — A UIViewController that embeds multiple Flutter instances.
+
+### Android
+
+## Data Synchronization Description
+
+This sample code uses a somewhat novel data synchronization mechanism to share
```

Comment:
nit: "somewhat novel" sounds a bit tentative/risky. The mechanism itself isn't that special. It's just a bit particular to work with the communication constrains present currently.

Reply:
I think just this

Reply:
done